invoices_app/views.py

```diff
@@ -274,7 +276,7 @@ def form_valid(self, form):
 
         if self.request.user.is_AP:
             user = self.request.user
-            _send_email_when_editing_invoice(form.instance, user)
+            # _send_email_when_editing_invoice(form.instance, user) ISSUE IN BACKLOG
```

Comment:
if you will do this I suggest you flag this line with a TODO comment. PyCharm has a feature to collect those comments https://www.jetbrains.com/help/pycharm/using-todo.html
But in general, is a bad practice push commented code to master